### DISCUSSION TITLE
v1.0.0 - First Stable Release!!!!! 🎉

### DISCUSSION THREAD

**Discussion by 0xCUB3** [2009-Nissan-Cube/About-This-Hack#57] [Announcements]
The biggest thing to happen to About This Hack since About This Hack...

## What's Changed
- Implemented a custom updater that is 2x faster and 10x lighter than the outgoing Sparkle framework
- Optimized the app, reducing loading times from 10 bounces to 2 (very scientific, I know)
- Simplified OpenCore output
- Added more detailed bootloader info
- Added nearly a dozen tooltips to view more hardware details
- Added metal version when applicable
- Reworked the entire storyboard for low-level optimizations
- Re-added support for macOS 10.12 Sierra
- Squashed over 200 bugs, including but not limited to Clover errors, data not loading, focus ring and button issues, initialization problems, memory leaks, and lag when switching views
- Probably like 100 more things, but I honestly forget at this point since so much has changed

*~~Please note that About This Hack is still in beta, so bugs exist. If you do find anything that isn't covered in the README, please [open an issue](https://github.com/0xCUB3/About-This-Hack/issues).~~* Haha not anymore, but feel free to keep making issues as they arise!

**Full Changelog**: https://github.com/0xCUB3/About-This-Hack/compare/0.9.1...1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/0xCUB3/About-This-Hack/releases/tag/1.0.0'>v1.0.0 - First Stable Release!!!!! 🎉</a>.</em>

**Comment by MDNich** [comment]
Wooooooo it's been a great few years working on this :)) glad to finally see it as a polished finished (but never final) product :)))

-MDNich
